thrimshim/thrimshim/main.py

```diff
@@ -155,6 +155,25 @@ def update_row(ident, new_row):
 	logging.info('Row {} updated to state {}'.format(ident, new_row['state']))
 	return ''
 
+@app.route('/thrimshim/reset/<ident>')
+def reset_row(ident):
+	"""Clear state and video_link columns and reset state to 'UNEDITED'."""
+	try:
+		uuid.UUID(ident)
+	except ValueError:
+		return 'Invalid format for id', 400
+	conn = app.db_manager.get_conn()
+	results = database.query(conn, """
+		UPDATE events 
+		SET STATE='UNEDITED', error = NULL, video_id = NULL, video_link = NULL,
+		uploader = NULL
```

Comment:
style nit: indent second line of the SET clause to make it clear it's still part of the SET.